manifests/init.pp

```diff
@@ -19,6 +19,7 @@
   Array[String] $activemq_hosts                = [ '127.0.0.1' ],
   Integer       $activemq_port                 = 8161,
   Boolean       $symlink_puppet_metrics_collector = true,
+  Optional[String] $influxdb_host                 = undef,
```

Comment:
Instead of influxdb_host I think this should be `Optional[Hash] influxdb_connection_info` or something similar.  Then we could collect all the various options the script accepts ( host, port, db name) without clogging up the parameters.  

In fact we could just do a `Hash $metrics_server_info` and then have the user fill in graphite, influx, or whatever the next option might be.